daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -389,7 +389,8 @@ packagingTests = testGroup "packaging"
             withCurrentDirectory tmpDir $
               withCreateProcess startProc $ \_ _ _ startPh -> do
               race_ (waitForProcess' startProc startPh) $ do
-                threadDelay 2000000
+                -- 20 secs for all the codegens to complete
```

Comment:
```suggestion
                -- 30 secs for all the codegens to complete
```

Reply:
reduced to 15 secs and changed the comment.